tests/test_dashboard.py

```diff
@@ -28,15 +28,21 @@ class TestDashboardEndpoints(unittest.TestCase):
     def setUp(self):
         global request, session, auth
         request = Request(globals()) # Use a clean Request object
+        request.folder = "/srv/web2py/applications/runestone/"
```

Comment:
Looks suspicious -- will only work in a Docker container...

Reply:
Yes, but I'm going to rewrite it entirely so I don't think it really matters.

Reply:
Sounds good.